Update manifest: ASIX.AX88179DriverSetup version 2.0.3.0

Dragon1573 opened the pull request:
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #314524

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/314544)

Dragon1573 committed:
Update manifest: ASIX.AX88179DriverSetup version 2.0.3.0

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

Dragon1573 committed:
Apply suggestions from code review

- Correction of installer hash value

stephengillie commented:
Automatic Validation ended with: 
> 
> Signature Update Failed.
> Failure to parse win 10, attempting win 11
> Installation failed with exit code -1978335141
> 2025-12-02 09:41:11.845 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\ASIX.AX88179DriverSetup.2.0.3.0\extracted\setup.exe 
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
  | 0000000B | 11 | -4294967285 | ERROR_BAD_FORMAT | An attempt was made to load a program with an incorrect format. | 
  | 8A15005B | -1978335141 | 2316632155 | APPINSTALLER_CLI_ERROR_NESTEDINSTALLER_NOT_FOUND | Failed to find the nested installer in the archive. Does the `RelativeFilePath` have the previous version number in it? | 


(Automated response - build 1166.)

Dragon1573 commented:
Can't get it ...

Most properties in the manifest hasn't changed ...

Trenly reviewed (changes requested):
The path inside the Zip file has changed

Dragon1573 committed:
Apply suggestions from code review

- See https://github.com/microsoft/winget-pkgs/pull/314544/changes#r2641859633
- Installer filename in archive has changed

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

Dragon1573 commented:
<img width="1666" height="644" alt="image" src="https://github.com/user-attachments/assets/13ec070e-a009-4e25-b64b-f45274252e2d" />
<img width="1720" height="908" alt="image" src="https://github.com/user-attachments/assets/29fd8be9-d63f-4fba-980d-87adf770e869" />

👀 Can't figure out why ...

If you try to install it in _Windows Sandbox_ interactively, no errors occur. Maybe it's caused by its InstallShield wizard?

Dragon1573 commented:
> [!NOTE]
> 
> There might be unknown reason that block this package from unattended installation. 
> 
> I'll give up this pull request ... 😞 
> 
> - See https://github.com/microsoft/winget-pkgs/pull/314544#issuecomment-3687461684
